features/polyExplorer/src/components/discoverCard/discoverCard.jsx

```diff
@@ -7,14 +7,17 @@ import "./discoverCard.css";
 const i18nHeader = "clusterStoriesPreview";
 const DiscoverCard = ({ story }) => {
     return (
-        <LinkButton route={story.route} className="preview">
+        <div className="discover-container">
             <img
                 src={story.img.src}
                 alt={i18n.t(`${i18nHeader}:${story.img.alt}`)}
             />
             <h3>{i18n.t(`${i18nHeader}:${story.title}`)}</h3>
             <p>{i18n.t(`${i18nHeader}:${story.previewText}`)}</p>
-        </LinkButton>
+            <LinkButton route={story.route} className="preview discover-button">
```

Comment:
In this case, wouldn't the two selectors `.discover-container .preview` and .discover-container .discover-button` collide? If you're not using either independently (probably not) maybe make them a single class?